### PR TITLE
cosmos-sdk v0.45.8

### PR DESCRIPTION
**Pull request by faddat**
## Description

Hi :D

Just a PR here that brings the liquidity module current with cosmos-sdk v0.45.8

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: x/liquidity/types/querier.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ const (
 
 // QueryLiquidityPoolParams is the query parameters for 'custom/liquidity'
 type QueryLiquidityPoolParams struct {
-	PoolId uint64 `json:"pool_id" yaml:"pool_id"` // nolint:golint
+	PoolId uint64 `json:"pool_id" yaml:"pool_id"` // nolint:revive
```
Comment (dongsam) [inline x/liquidity/types/querier.go:14]: I think It should be `//nolint:revive`

Comment (kogisin) [reply x/liquidity/types/querier.go:14]: `Error: directive `// nolint:revive` should be written without leading space as `//nolint:revive` (nolintlint)`

**Review by kogisin — COMMENTED** [review]
LGTM other than the failing lint check.

**Comment by kogisin** [comment]
Hey @faddat, it seems like this PR doesn't have permission granted for maintainer(s) to directly commit changes. As I test to fix the lint issue locally, I found the following issues that can be resolved. You'll be greatly appreciated if you make the changes. Thanks in advance!

- Remove the linter `deadcode` in [golangci.yml](https://github.com/notional-labs/liquidity/blob/457/.golangci.yml#L10) as it is deprecated and replaced by `unused`.
- Fix `// nolint:revive` to `//nolint:revive (no space in between)`

**Comment by faddat** [comment]
totally!

Thanks for checking thorough this.

**Comment by faddat** [comment]
Random question:

When I make PR's from notional-labs repos, people cannot update them as you describe.  When I make them as me, they can.

Do you have any idea how we can fix that?

**Comment by kogisin** [comment]
Did you allow "Allow edits from maintainers" check box when you open this PR? If you do, then owners of a project can directly push their commits to your PR.

**Pull request merged**